docs/self-managed/components/orchestration-cluster/identity/configuration.md

```diff
@@ -78,19 +78,19 @@ The following variables are used to initialize users and mapping rules.
-| `camunda.security.initialization.users[0].email`                      | The email address of the first user.         |               |
-| `camunda.security.initialization.mappingRules[0].mappingRuleId`       | The id of the first mapping rule.            |               |
-| `camunda.security.initialization.mappingRules[0].claimName`           | The claim of the first mapping rule.         |               |
-| `camunda.security.initialization.mappingRules[0].claimValue`          | The claim's value of the first mapping rule. |               |
-| `camunda.security.initialization.defaultRoles.<role>.users[0]`        | Users assigned to the `<role>` role.         |               |
-| `camunda.security.initialization.defaultRoles.<role>.groups[0]`       | Groups assigned to the `<role>` role.        |               |
-| `camunda.security.initialization.defaultRoles.<role>.clients[0]`      | Clients assigned to the `<role>` role.       |               |
-| `camunda.security.initialization.defaultRoles.<role>.mappingRules[0]` | Mapping rules assigned to the `<role>` role. |               |
+| Application.yaml property                                               | Description                                  | Default value |
+| ----------------------------------------------------------------------- | -------------------------------------------- | ------------- |
+| `camunda.security.initialization.users[0].username`                     | The username of the first user.              |               |
+| `camunda.security.initialization.users[0].password`                     | The password of the first user.              |               |
+| `camunda.security.initialization.users[0].name`                         | The name of the first user.                  |               |
+| `camunda.security.initialization.users[0].email`                        | The email address of the first user.         |               |
+| `camunda.security.initialization.mapping-rules[0].mapping-rule-id`      | The id of the first mapping rule.            |               |
+| `camunda.security.initialization.mapping-rules[0].claim-name`           | The claim of the first mapping rule.         |               |
+| `camunda.security.initialization.mapping-rules[0].claim-value`          | The claim's value of the first mapping rule. |               |
+| `camunda.security.initialization.default-roles.<role>.users[0]`         | Users assigned to the `<role>` role.         |               |
+| `camunda.security.initialization.default-roles.<role>.groups[0]`        | Groups assigned to the `<role>` role.        |               |
+| `camunda.security.initialization.default-roles.<role>.clients[0]`       | Clients assigned to the `<role>` role.       |               |
+| `camunda.security.initialization.default-roles.<role>.mapping-rules[0]` | Mapping rules assigned to the `<role>` role. |               |
```

Comment:
The kebab-case is not supported here, as it's not managed by spring properties and we are handling it in application dynamically: [IdentitySetupInitializer.java](https://github.com/camunda/camunda/blob/e7c723d83216e6943bf5ebb03faa02bf49e55a15/zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/user/IdentitySetupInitializer.java#L126)
The accepted values can be `mappingRules` or `mappingrules`.